build.gradle

```diff
@@ -32,19 +32,27 @@ if (System.getenv('CI') == 'true') {
   gradle.startParameter.showStacktrace = ShowStacktrace.ALWAYS
 }
 
-import org.ajoberstar.grgit.Grgit
-
 allprojects {
-  apply plugin: 'org.ajoberstar.grgit'
-  def git = Grgit.open(currentDir: rootDir)
-  def maybeSnapshot = System.getenv('CIRCLE_BRANCH') == 'master' ? '' : '-SNAPSHOT'
-  def shortCommitHash = "+git.${git.head().abbreviatedId}"
-  def maybeDirty = git.status().clean ? '' : '-dirty'
-  git.close()
-
   group 'com.virtuslab'
+
   apply from: "$rootDir/version.gradle"
-  version "${PLUGIN_VERSION}${maybeSnapshot}${shortCommitHash}${maybeDirty}"
+  def ciBranch = System.getenv('CIRCLE_BRANCH')
+  if (ciBranch == 'master') {
+    version PLUGIN_VERSION
+  } else {
+    // As for now, we're unfortunately not running builds for develop yet
```

Comment:
_Slack discussion summary_

Let's consider running CI builds for all branches.

Reply:
Done, reconfigured CI, updating the comment...